2015-11-20 15:54:13,662 - [INFO] - from play in pool-4-thread-1 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2015-11-20 15:57:57,197 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 16:10:55,189 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 16:19:56,550 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 16:21:28,154 - [ERROR] - from application in New I/O worker #5 


! @6o7enb48f - Internal server error, for (GET) [/] ->

play.PlayExceptions$RoutesCompilationException: Compilation error[Missing parameter in call definition: postId]
	at play.PlaySourceGenerators$$anonfun$1.apply(PlaySourceGenerators.scala:24) ~[na:na]
	at play.PlaySourceGenerators$$anonfun$1.apply(PlaySourceGenerators.scala:12) ~[na:na]
	at play.PlaySettings$$anonfun$defaultSettings$29.apply(PlaySettings.scala:119) ~[na:na]
	at play.PlaySettings$$anonfun$defaultSettings$29.apply(PlaySettings.scala:119) ~[na:na]
	at scala.Function7$$anonfun$tupled$1.apply(Function7.scala:35) ~[scala-library-2.11.4.jar:na]
	at scala.Function7$$anonfun$tupled$1.apply(Function7.scala:34) ~[scala-library-2.11.4.jar:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[scala-library-2.11.4.jar:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_60]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2015-11-20 16:22:03,560 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 16:31:14,991 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 16:38:11,661 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 16:44:49,268 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 16:48:04,244 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 16:50:52,598 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 16:54:10,030 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 17:15:15,502 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 17:30:03,693 - [ERROR] - from application in New I/O worker #3 


! @6o7f6b8a0 - Internal server error, for (GET) [/] ->

play.PlayExceptions$RoutesCompilationException: Compilation error[Missing parameter in call definition: followId]
	at play.PlaySourceGenerators$$anonfun$1.apply(PlaySourceGenerators.scala:24) ~[na:na]
	at play.PlaySourceGenerators$$anonfun$1.apply(PlaySourceGenerators.scala:12) ~[na:na]
	at play.PlaySettings$$anonfun$defaultSettings$29.apply(PlaySettings.scala:119) ~[na:na]
	at play.PlaySettings$$anonfun$defaultSettings$29.apply(PlaySettings.scala:119) ~[na:na]
	at scala.Function7$$anonfun$tupled$1.apply(Function7.scala:35) ~[scala-library-2.11.4.jar:na]
	at scala.Function7$$anonfun$tupled$1.apply(Function7.scala:34) ~[scala-library-2.11.4.jar:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[scala-library-2.11.4.jar:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_60]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2015-11-20 17:30:38,172 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 17:55:14,778 - [ERROR] - from application in New I/O worker #5 


! @6o7f9j7lg - Internal server error, for (GET) [/users/1] ->

play.PlayExceptions$CompilationException: Compilation error[error: cannot find symbol]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2015-11-20 17:55:16,014 - [ERROR] - from application in New I/O worker #1 


! @6o7f9j9h7 - Internal server error, for (GET) [/users/1] ->

play.PlayExceptions$CompilationException: Compilation error[error: cannot find symbol]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2015-11-20 17:57:13,451 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

